assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -37,11 +37,22 @@ type RegularContribution = {|
   billingPeriod: BillingPeriod,
 |};
 
-type PayPalDetails = {| baid: string |};
+// TODO: can we do away with these types and use the PaymentAuthorisation here?
+// and thus do away with getPaymentFields and paymentDetailsFromAuthorisation
+// (would probably require backend renaming)
```

Comment:
Yeah, I would keep it that way until then, at least it is "type-safe". A generic `PaymentAuthorisation => JSON` could introduce bugs.